### ISSUE TITLE
Recently uploaded packages not available for install

### ISSUE THREAD

**Issue by AArnott** [NuGet/Home#889]
I just uploaded a couple packages:
https://www.nuget.org/packages/ImmutableObjectGraph.Generation/1.0.15185-beta6
https://www.nuget.org/packages/ImmutableObjectGraph/1.0.15185-beta6

I can clearly see them online. But when I try to install them into a VS project, I’m told they’re not found. 

> PM> Install-Package ImmutableObjectGraph.Generation -Pre -version 1.0.15185-beta6
Attempting to gather dependencies information for package 'ImmutableObjectGraph.Generation.1.0.15185-beta6' with respect to project 'ConsoleApplication4', targeting '.NETFramework,Version=v4.5.2'
Install-Package : Package 'ImmutableObjectGraph.Generation' is not found
At line:1 char:1
+ Install-Package ImmutableObjectGraph.Generation -Pre -version 1.0.151 ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 

    PM> Find-Package immutableobjectgraph -pre
    Id                                  Versions                                 Description                                                  
    --                                  --------                                 -----------                                                  
    ImmutableObjectGraph                {1.0.0-beta5-gcca633c3de}                Runtime library for immutable types                          
    ImmutableObjectGraph.T4             {0.1.0-beta4}                            T4 templates for creating immutable types                    
    ImmutableObjectGraph.Core           {0.1.0-beta4}                            Runtime library for immutable types                          
    ImmutableObjectGraph.Generation     {1.0.0-beta5-gcca633c3de}                Roslyn-based code generation for immutable types             

It's been a couple hours, and I don't see any notice on the web site saying indexing is still in progress.

**Comment by yishaigalatzer** [comment]
The packages are now available as far as I can tell. Will follow up on the delay and why it didn't who up.

This kind of issues should go to the gallery repo (and consider sending a support email if you are blocked, bugs are not looked at a high enough priority).

**Comment by yishaigalatzer** [comment]
1.	The V3 indexing job has been lagging behind catalog by around 30-40 minutes. This was because the indexing jobs used the CDN url (https://api.nuget.org/v3-catalog0/index.json) to access Catalog and due to CDN caching, the Catalog2Registration and Catalog2Lucene jobs might still see an old commit stamp in catalog and assume that the catalog hasn’t been changed while new packages might have been added to the catalog. We changed the these jobs and relevant monitoring jobs to use the blob storage Url directly instead of CDN url to avoid the delay. The change is made in int and will get promoted to prod today as part of the v2/v3 merge job deployment.
2.	The comment about not being able to see “index in progress” details : that is because Gallery shows indexing details based on V2 Lucene whereas he is trying to install packages via V3 ( which uses V3 Lucene). This inconsistency should go away once we have v2/v3 merge.
